src-vue/src/plugins/store.ts

```diff
@@ -226,21 +234,51 @@ async function _initializeApp(state: any) {
     // Get FlightCore version number
     state.flightcore_version = await invoke("get_flightcore_version_number");
 
-    const result = await invoke("find_game_install_location_caller")
-        .catch((err) => {
-            // Gamepath not found or other error
-            console.error(err);
-            notification_handle = ElNotification({
-                title: 'Titanfall2 not found!',
-                message: "Please manually select install location",
-                type: 'error',
-                position: 'bottom-right',
-                duration: 0 // Duration `0` means the notification will not auto-vanish
+    var result = undefined;
+    var persistent_game_install = (await persistentStore.get('game-install')) as any;
+    console.log(persistent_game_install);
```

Comment:
Don't forget to remove this :)

Reply:
Removed with aaf142767ea36bdbc87345612e64ea6610999190 ^^